src/drivers/imx/CMakeLists.txt

```diff
@@ -15,3 +14,7 @@ endif()
 if(CONFIG_IMX_EDMA)
 	add_local_sources(sof edma.c)
 endif()
+
+if(CONFIG_IMX_SDMA)
+	add_local_sources(sof sdma.c)
+endif()
```

Comment:
I think we need to do the other way around. CONFIG_IMX_SDMA should be placed under src/drivers/imx/Kconfig as SDMA, EDMA, IRQSTEER are imx specific config options

Reply:
I'll add a new commit that moves them around into a newly created Kconfig file.

Reply:
@paulstelian97 so to make things clear. The newly added commit should be the one adding the newly introduced config symbols directly.

Do not add them into src/drivers/Kconfig and then move them to src/drivers/imx/Kconfig. Just directly add them
in the right place (this is src/drivers/imx/Kconfig).

Reply:
Alright, I will rearrange them after my lunch break.